rs/p2p/quic_transport/BUILD.bazel

```diff
@@ -62,6 +64,14 @@ rust_library(
     deps = DEPENDENCIES,
 )
 
+rust_test(
+    name = "quic_transport_unit",
```

Comment:
```suggestion
    name = "quic_transport_test",
```

Reply:
Seems like we use the "unit" and "integration" suffix for most of our tests. Wouldn't this break with that convention?

Reply:
I thought our convention is "test" and "integration" (which transforms to "integration_test")